components/src/dynamo/frontend/tests/test_multimodal_utils.py

```
@@ -162,4 +174,114 @@ def test_handles_malformed_content_non_dict():
+
+
+@pytest.mark.parametrize("part_type", ["audio_url", "video_url"])
+@pytest.mark.parametrize("nested", [False, True])
+def test_rejects_audio_video_cache_uuids(part_type: str, nested: bool) -> None:
+    media = {"url": "https://example.com/media", "uuid": "cached-media"}
+    part = {"type": part_type, part_type: media}
+    if not nested:
+        media.pop("uuid")
+        part["uuid"] = "cached-media"
+
+    with pytest.raises(ValueError, match="supported only for image_url.*vLLM"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)